Projects/BookJSON2/main.go

```diff
@@ -30,26 +31,39 @@ var books []Book
 func (b *Books) book(w http.ResponseWriter, r *http.Request) {
 	// if r.Method == POST
 	switch r.Method {
-	case "POST": //Retriving a book with Id number
+	case "GET": //Retriving a book with Id number
```

Comment:
Go already has enumerations for these literals

https://github.com/golang/go/blob/2a78e8afc0994f5b292bc9a5a7258c749e43032f/src/net/http/method.go#L11-L19

```suggestion
	case http.MethodGet:
```

Reply:
resolved with commit https://github.com/AmanSarraf/Go-Project/pull/9/commits/85fdee5b41960dddf4790f50c6def83cb4fe3ea5